extensions/ql-vscode/src/codeql-cli/cli.ts

```diff
@@ -1417,6 +1420,27 @@ export class CodeQLCliServer implements Disposable {
     );
   }
 
+  public async packCreate(
```

Comment:
I don't see any usages of this. Is that intentional?

Reply:
No, I think I must have added it before I realized we actually needed to call `pack bundle` instead. Removed.